analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    la_nc = gpd.read_file(os.path.join(dir, 'Neighborhood_Councils_(Certified)/Neighborhood_Councils_(Certified).shp'))
</pre><p>Please rename these files to remove parentheses</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. </p>